charts/athens-proxy/values.yaml

```diff
@@ -47,6 +48,12 @@ storage:
 netrc:
   # if enabled, it expects to find the content of a valid .netrc file imported as a secret named netrcsecret
   enabled: false
+  existingSecret: netrcsecret
+
+goCenter:
```

Comment:
@rimusz I think this section is more general, since you can set any URL you'd like. So could we rename the section name to "upstream"?